VxWorks/Os/Queue.cpp

```diff
@@ -91,9 +93,13 @@ FwSizeType VxWorksQueue::getMessagesAvailable() const {
 }
 
 FwSizeType VxWorksQueue::getMessageHighWaterMark() const {
+#ifdef ENABLE_HIGH_WATERMARK
     // Safe to cast away const in this context because scope lock will restore unlocked state on return
     Os::ScopeLock lock(const_cast<Mutex&>(this->m_handle.m_data_lock));
     return this->m_handle.m_highMark;
+#else
+    return 0;
+#endif
```

Comment:
Delete this block and replace it with

```c++
return std::numeric_limits<FwSizeType>::max();
```
and add this to the top of this file
```C++
#include <limits>
```

Reply:
Also, update VxWorks/Os/Queue.hpp comment for this function and update the "brief" description to indicate this function is not supported. And update the comment about return to indicate it returns the max value of FwSizeType.